.github/workflows/ci_cd.yml

```
@@ -64,9 +64,6 @@ jobs:
         username: ${{ secrets.USERNAME }}
         key: ${{ secrets.SSH_KEY }}
         script: |
-          cd build
-          git -C HTTP-Server pull || git clone git@github.com:ksdenisova/HTTP-Server.git
-          cd HTTP-Server
-          kill -9 $(lsof -t -i:5000)
-          mix deps.get
-          elixir --erl "-detached" -S mix run --no-halt
+          sh build/HTTP-Server/scripts/deploy.sh 
+          sh build/HTTP-Server/scripts/stop.sh
+          sh build/HTTP-Server/scripts/start.sh
```

this looks good, I would suggest this order:

```suggestion
          sh build/HTTP-Server/scripts/stop.sh
          sh build/HTTP-Server/scripts/deploy.sh 
          sh build/HTTP-Server/scripts/start.sh
```

---

Agree, I'll make the changes.